dsfs/dataset_test.go

```diff
@@ -216,13 +214,13 @@ func TestCreateDataset(t *testing.T) {
 		t.Errorf("case nil datafile and no PreviousPath, error mismatch: expected '%s', got '%s'", expectedErr, err.Error())
 	}
 	// take path from previous case
-	ds.PreviousPath = cases[2].resultPath
-	_, err = CreateDataset(store, ds, nil, privKey, false)
-	if err != nil {
-		t.Errorf("case nil datafile and PreviousPath, error mismatch: expected no error, got '%s'", err.Error())
-	}
-	if len(store.(memfs.MapStore)) != 18 {
-		t.Errorf("case nil datafile and PreviousPath, expected invalid number of entries: %d != %d", 18, len(store.(memfs.MapStore)))
+	// ds.PreviousPath = cases[2].resultPath
```

Comment:
is this test no longer using previous path? are we ok to delete this?

Reply:
just pushed a commit that updates this